tests/queries/0_stateless/02813_merges_with_partition_mutations.sql.j2

```diff
@@ -0,0 +1,55 @@
+DROP TABLE IF EXISTS multipart;
+CREATE TABLE multipart
+(
+    `n` UInt32,
+    `p` UInt32,
+    `s` String
+)
+ENGINE = MergeTree
+PARTITION BY p
+ORDER BY n;
+
+{% macro WATCH(message) -%}
+select '{{ message }}';
+select name from system.parts where table='multipart' and database=currentDatabase() and active=1 and (name LIKE '7\\_%' OR name LIKE '8\\_%');
```

Comment:
`OR name LIKE '8\\_%` why? 

Maybe add `partition_id` as macro parameter? So you don't accidentally get info about partition=8 while modifying partition=7 or vice versa?